<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-M0Q06O0F" failures="1" tests="5" name="Test" time="0.069" errors="0" timestamp="2022-01-05T14:39:12 IST">
  <testcase classname="com.testingannotation.SampleRetry" name="usernamevalidation" time="0.008">
    <skipped/>
  </testcase> <!-- usernamevalidation -->
  <testcase classname="com.testingannotation.SampleRetry" name="usernamevalidation" time="0.001">
    <skipped/>
  </testcase> <!-- usernamevalidation -->
  <testcase classname="com.testingannotation.SampleRetry" name="usernamevalidation" time="0.001">
    <skipped/>
  </testcase> <!-- usernamevalidation -->
  <testcase classname="com.testingannotation.SampleRetry" name="usernamevalidation" time="0.001">
    <skipped/>
  </testcase> <!-- usernamevalidation -->
  <testcase classname="com.testingannotation.SampleRetry" name="usernamevalidation" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [HARESH] but found [haresh]">
      <![CDATA[java.lang.AssertionError: expected [HARESH] but found [haresh]
at com.testingannotation.SampleRetry.usernamevalidation(SampleRetry.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- usernamevalidation -->
</testsuite> <!-- Test -->
